exporter/clickhouseexporter/README.md

```diff
@@ -25,16 +25,16 @@ Support time-series graph, table and logs.
 ```clickhouse
 /* get error count about my service last 1 hour.*/
 SELECT count(*)
-FROM logs
```

Comment:
Why `logs_local`? If we stick to Otel data model, maybe better to call it `otel_logs`? Also wouldn't user be interested to use a custom table name?

Reply:
Make sense.

Reply:
OK. i will update it today. :)

Reply:
Update done.